Source/WebCore/html/FormController.cpp

```diff
@@ -267,14 +267,14 @@ Vector<String> FormController::formElementsState(const Document& document) const
         return a.control->insertionIndex() < b.control->insertionIndex();
     });
 
-    Vector<String> stateVector;
+    Vector<AtomString> stateVector;
     stateVector.append(formStateSignature());
     for (size_t i = 0, size = controls.size(); i < size; ) {
         auto formStart = i;
         auto formKey = controls[formStart].formKey;
         while (++i < size && controls[i].formKey == formKey) { }
-        stateVector.append(formKey);
-        stateVector.append(String::number(i - formStart));
+        stateVector.append(AtomString { formKey });
+        stateVector.append(AtomString::number(i - formStart));
```

Comment:
We should probably delete `AtomString::number` and `String::number` and just use `makeAtomString` and `makeString`, presuming we can approve they have identical or better efficiency when passed a single numeric argument.